client/src/components/QuestionsAnswers.jsx

```diff
@@ -1,15 +1,34 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
 import QuestionsSearch from './Sub_Questions/QuestionSearch.jsx';
 import QuestionList from './Sub_Questions/QuestionList.jsx';
 import AskQuestion from './Sub_Questions/AskQuestion.jsx';
 import AnswerQuestion from './Sub_Questions/AnswerQuestion.jsx';
 
 export default function QuestionsAnswers() {
+  const tempId = 40355;
```

Comment:
on line 9it seems that you hardcoded the productID. I'm wondering why not use the productID passed from the App component.

Reply:
oh this is just because this product had more populated questions and answers